velox/expression/Expr.cpp

```diff
@@ -193,7 +193,10 @@ void Expr::computeMetadata() {
   }
 
   for (auto& input : inputs_) {
-    input->computeMetadata();
+    if (!input->isMultiplyReferenced_) {
+      // These should already have metadata computed.
```

Comment:
This comment seems to contradict the code. If metadata is already computed, why compute it again?

Reply:
thanks for catching this, its an incomplete comment. What I meant to say was that isMultiplyReferenced expression would have metadata computed so we can skip those. Will update the comment appropriately.

Reply:
Also FYI, this skipping is not necessary for the fix, but just an optimization to avoid redundant work for something that already is computed.

Reply:
Thank you for clarifying. Perhaps, mention this in the PR description.